mainsite/catalog/views.py

```diff
@@ -85,6 +97,9 @@ def search(request):
 #returns: all the items in the database, with the most recently item added at the top
 @login_required(login_url='/')
 def index(request):
+    if (isCurrentlyBanned(request.user)):
```

Comment:
It might make more sense to use a view decorator instead of checks inside the function body.

I.e.: https://docs.djangoproject.com/en/1.10/topics/auth/default/#django.contrib.auth.decorators.user_passes_test

Reply:
Yep your right.